queue.go

```diff
@@ -0,0 +1,36 @@
+package queue
+
+import (
+	"fmt"
+)
+
+type Song string

```

Comment:
change this to SongUUID, make a song struct because we will want to add more stuff later